src/dllmain.cpp

```diff
@@ -102,6 +102,8 @@ void Shutdown()
 }
 
 BOOL APIENTRY DllMain(HMODULE mod, DWORD ul_reason_for_call, LPVOID) {
+	DisableThreadLibraryCalls(mod);
```

Comment:
indentation looks wrong

Reply:
no, is right here... problem is that { should be on next line, following your style. 

corrected.